tests/core/test_agent.py

```diff
@@ -163,6 +163,18 @@ def test_agent_wrong_use_of_load(tmpdir, default_domain):
     assert agent.model_directory is not None
 
 
+async def test_agent_update_model_none_domain(trained_model):
+    agent = await load_agent(model_path=trained_model)
+    agent.update_model(
+        None,
+        agent.policy_ensemble,
```

Comment:
this would in practice be `None` as well in the case of no Core model (i.e. no domain)